src/class.js

```diff
@@ -496,3 +500,19 @@ Fire._fastDefine = function (className, constructor, serializableFields) {
         Fire.attr(constructor, serializableFields[i], Fire.HideInInspector);
     }
 };
+
+// @ifdef DEV
+/**
+ * check Ctor is a function and is no parameters
+ */
+function _checkCtor(ctor) {
+    if (typeof ctor !== 'function') {
+        Fire.warn("构造函数必须是方法");
+        return;
+    }
+    if (ctor.length > 0) {
+        Fire.warn("为了规范 FireClass 的序列化操作，实例化过程中不允许依赖构造参数。如果专家用户确实有需要，可以从 arguments 中获取传入参数");
+        return;
```

Comment:
// To make a unified FireClass serialization process, we don't allow parameters for constructor when creating instances of FireClass.
// For advance user, construct arguments can be rechieved from 'arguments'.
Fire.warn('Can not instantiate FireClass with arguments.');